setup.py

```diff
@@ -31,15 +31,17 @@
       'black'
     },
     'local': [
-      'pydotplus==2.0.2',
-      'adage==0.10.1',
-      'yadage-schemas==0.10.6',
-      'packtivity==0.14.23',
-      'yadage[viz]==0.20.1',
```

Comment:
I'm not sure how important it is for these to be pinned, but we can resolve this after PR #44 is in.